server/data/reviews.js

```diff
@@ -89,10 +88,33 @@ const exportedMethods = {
       };
     const insertInfo = await reviewCollection.insertOne(newReview);
     if (insertInfo.insertedCount === 0) throw 'Could not add review';
-    console.log(insertInfo);
-    return "Add review successfully";
-  
+    return insertInfo;
+    },
+
+    async AddReviewToUser(userid,reviewId){
+      const userCollection = await users();
```

Comment:
can we do an error check on userid and review id, like if they exist, isvalidobjectid and all